src/lib/models/sml-versions.ts

```diff
@@ -1,25 +1,41 @@
 import * as zod from 'zod';
 import type { VersionStabilities } from '$lib/generated';
 
+export type SMLLinkData = {
```

Comment:
Please change to `SMLArchData`

Reply:
https://github.com/satisfactorymodding/smr-frontend/pull/47/commits/9c0893e0353c97b347f9f0152ec590c614d255ac